src/components/Icon/Icon.js

```diff
@@ -14,7 +15,7 @@ export default function Icon ({
   const iconClassName = `icon-${name}`
 
   return <span
-    styleName={cx('icon', { green })}
+    styleName={cx('icon', { green, blue })}
```

Comment:
how can an icon be both green and blue?

Reply:
Those are props. But if someone passed in `green = true` and `blue = true`, then it would apply both classes but that would be obviously wrong for them to do that.

Reply:
ah yes right